appengine/standard_python37/django/requirements.txt

```diff
@@ -0,0 +1,2 @@
+Django==2.1.1
+mysqlclient==1.3.13
```

Comment:
Can we use pymysql here instead? We're trying to standardize on that across our samples as complaints about installing mysql dev libs are very common.

Reply:
Got it. Done.